tests/ha/sle15_workarounds.pm

```diff
@@ -0,0 +1,49 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Add some SLE15 workarounds
+#          Should be removed after SLE15 will be released!
+# Maintainer: Loic Devulder <ldevulder@suse.com>
+
+use base "opensusebasetest";
+use strict;
+use utils;
+use testapi;
+use hacluster;
+
+# Install missing patterns/packages for HA support in SLE15
+sub run {
+    return unless sle_version_at_least('15');
+
+    # Modify the device number if needed
+    if (get_var('ISO_1', '') ne '') {
+        assert_script_run "sed -i 's;sr1;sr0;g' /etc/zypp/repos.d/*";
+    }
+
+    # Install ha_sles pattern if missing
+    if (script_run 'zypper pt -i | grep -q ha_sles') {
+        record_soft_failure 'bsc#1060472';
+        zypper_call("in -l -t pattern ha_sles", exitcode => [0, 102], timeout => 900);
+    }
+
+    # Install ocfs2-kmp-default package if missing
+    if (script_run 'rpm -q ocfs2-kmp-default >/dev/null') {
+        record_soft_failure 'bsc#1060601';
```

Comment:
the last comment in the bug states that the bug is probably fixed, please verify

Reply:
It was already here when I retrieved the HA tests. But yes it is not HA related, so I could easily put it in another place.
I find an already existing NTP client configuration in tests/console/yast2_ntpclient.pm, but not sure that I can use it (HA tests NTP sync on support_server).

I could create a poo# for this and work on it after, is it ok for you?

Reply:
I checked on the last run tests and yes it has been corrected, so I will remove it.